doc/Roadmap.md

```diff
@@ -7,18 +7,17 @@ Please feel free to comment!
 * Push notifications:
     * [XEP-0357: Push Notifications](https://xmpp.org/extensions/xep-0357.html)
     * Amazon AWS SNS (Simple Notification Service)
-* Instant Stream Resumption
 * Full text search in archive
-* Bind 2.0 (experimental)
```

Comment:
This is still valid. Bind 2.0 is not in beta1, it doesn't mean it will not fit into final 2.1